lib/models/page.js

```diff
@@ -698,6 +713,25 @@ module.exports = function(crowi) {
     return this.findListByStartWith(path, userData, { limit: 0, ...option })
   }
 
+  pageSchema.statics.findUnfurlablePages = async function(type, array) {
+    const Page = this
+    const page = await Page.find({
+      [type]: { $in: array },
+      $or: [{ grant: GRANT_PUBLIC }, { grant: GRANT_RESTRICTED }],
```

Comment:
We can't view `GRANT_RESTRICTED` pages without knowing id currently.
I'm worried about someone can explore pages by guessing page path without regard of author's intent.

Reply:
Thank you!
I was misunderstanding the specification.